src/components/Home/Index.jsx

```diff
@@ -122,6 +123,96 @@ function Index() {
           </div>
         </div>
       </Section>
+
+      <Row id="about" className="justify-content-center">
```

Comment:
One aspect I don't like so much is that when we link to this, the section caption is not visible. I believe this is due to the height of our header obscuring a part of the page not being considered when jumping to the anchor. Any idea that currently comes to my mind on how to fix it would be an ugly hack, so I'm inclined to accept it for now until we have a good idea on how to solve this.